docs/DafnyRef/Types.md

```diff
@@ -430,16 +430,30 @@ OrdinalType_ = "ORDINAL"
 ````
 
 Values of type `ORDINAL` behave like `nat`s in many ways, with one important difference:
-there is an `ORDINAL` value that is larger than any `nat`.
-- a value of type `nat` may be explicitly converted to an `ORDINAL` using `as ORDINAL`
-- a value of type `ORDINAL` may be explicitly converted to a `nat` using `as nat` if it is known to be less than some `nat` value 
-- non-negative numeric literals may be considered `ORDINAL` literals
+there are `ORDINAL` values that are larger than any `nat`. The smallest of these non-nat ordinals is
+represented as $\omega$ in mathematics, though there is no literal expression in Dafny that represents this value.
```

Comment:
Except maybe; `var w :| !w.IsNat && (forall z:: z.IsNat || w <= z);` ?

Reply:
I thought of that as well, but  actually meant a Literal, like 5 or true. Though I think it would be
var w :| !w.IsNat && (forall z:: !z.IsNat || w <= z)